### ISSUE TITLE
.deb not creating user

### ISSUE THREAD

**Issue by Metigoth** [advplyr/audiobookshelf-ppa#1]
.deb is not creating user/group audiobookshelf.

Workaround create user audiobookshelf.

**Comment by advplyr** [comment]
Did you spot any errors after the install? The user and group are created in the post install script: https://github.com/advplyr/audiobookshelf/blob/master/build/debian/DEBIAN/postinst

You should see "Creating system user" and "Creating system group"

**Comment by Metigoth** [comment]
ffmpeg-git-20220108-amd64-static/ffmpeg
chown: invalid user: ‘audiobookshelf:audiobookshelf’
dpkg: error processing archive /home/user/audiobookshelf_1.6.58_amd64.deb (--unpack):
 new audiobookshelf package pre-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 /home/user/audiobookshelf_1.6.58_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Comment by Metigoth** [comment]
Uninstall leaves file behind
Removing audiobookshelf (1.6.58) ...
Removed /etc/systemd/system/multi-user.target.wants/audiobookshelf.service.
dpkg: warning: while removing audiobookshelf, directory '/usr/lib/audiobookshelf-ffmpeg' not empty so not removed
dpkg: warning: while removing audiobookshelf, directory '/usr/share/audiobookshelf' not empty so not removed

**Comment by advplyr** [comment]
I fixed the issue with the user/group not getting created. Latest deb is available `v1.6.59`

For the uninstall I attempted to setup a postrm script to remove audiobookshelf-ffmpeg, but it was causing an issue when upgrading so I didn't include it.
I don't think the entire directory /usr/share/audiobookshelf should be removed because it will contain the database, metadata and audiobooks by default. There is probably a best practice here that I'm missing.
